posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,90 @@
+
+
+== Workshops at JCON 2021
+
+If you're interested in attending the 'Big workshop day' on Friday 8th:
+
+* https://jcon.sched.com/event/k5do/cloud-native-java-made-easy-with-microprofile-and-jakarta-ee?iframe=no[Cloud-Native Made easy with MicroProfile and Jakarta EE]
++
+https://twitter.com/gracejansen27[Grace Jansen], https://twitter.com/Jamie_Lee_C[Jamie Coleman], https://twitter.com/yeekangc[YK Chang] and https://twitter.com/mgrygles[Mary Grygleski] will lead this workshop where you can experience first-hand how to build cloud-native solutions quickly and efficiently with open enterprise-grade cloud-native Java programming APIs optimized for microservices and the cloud. They'll be covering a range of topics in a hands-on-manner, including: developing RESTful and reactive services, effective testing, and application considerations for cloud-native applications. 
+
+* https://jcon.sched.com/event/kK4C/running-your-java-enterprise-on-open-liberty-on-azure-red-hat-openshift?iframe=no[Running your Java enterprise on Open Liberty on Azure Red Hat OpenShift]
++
+https://twitter.com/gcharters[Graham Charters] will team up with Microsoft's Principal Architect https://twitter.com/edburns[Edward Burns] to lead this fast-paced hands-on workshop, where you'll see first-hand how to run Java EE, Jakarta EE and MicroProfile applications on Open Liberty on Azure Red Hat OpenShift (ARO). The combination of ARO with Open Liberty offers a powerful and flexible platform for your enterprise Java applications. 
+
+
+
+=== Here's a couple of talks about MicroProfile and Jakarta EE by others contributing to the these fantastic open-source projects:
+
+* https://jcon.sched.com/event/jzz3/jakarta-ee-core-profile-a-slimmer-jakarta-ee?iframe=no[Jakarta EE Core Profile: A slimmer Jakarta EE] by https://twitter.com/ivar_grimstad[Ivar Grimstad] (Jakarta EE Developer Advocate, Eclipse Foundation)
+
+* https://jcon.sched.com/event/k3Lz/jakarta-ee-security-sailing-safe-in-troubled-waters?iframe=no[Jakarta EE Security - Sailing Safe in Troubled Waters] by https://twitter.com/ivar_grimstad[Ivar Grimstad] (Jakarta EE Developer Advocate, Eclipse Foundation)
+
+* https://jcon.sched.com/event/jwVc/a-freakonomic-take-on-open-standards-and-jakarta-ee?iframe=no[A freakonomic take on open standard and Jakarta EE] by https://twitter.com/reza_rahman[Reza Rahman] (Principal Program Manager, Microsoft)
+
+* https://jcon.sched.com/event/k5dw/effective-kubernetes-for-jakarta-ee-and-microprofile-developers?iframe=no[Effective Kubernetes for Jakarta EE and MicroProfile Developers] by https://twitter.com/reza_rahman[Reza Rahman] (Principal Program Manager, Microsoft)
+
+
+
+And finally, for those intersted in talks in other languages on these topics, check out the following session:
+
+* https://jcon.sched.com/event/k3La/dream-team-jakarta-ee-microprofile?iframe=no[Dream-Team Jakarta EE + MicroProfile] by https://twitter.com/dirkweil?lang=en[Dirk Weil] (CEO, GEDOPLAN GmbH)
```

Comment:
consider mentioning what specific language the talk is in, rather than just "other"